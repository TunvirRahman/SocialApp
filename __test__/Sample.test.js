
import 'react-native'
import React from 'react';
import app from './../App';
import renderer from 'react-test-renderer';

it('testRender',()=>{
const tree = renderer.create(
<app></app>
).toJSON();
expect(tree).toMatchSnapshot();
})