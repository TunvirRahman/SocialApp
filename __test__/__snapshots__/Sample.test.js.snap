// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testRender 1`] = `<app />`;
